lib/ansible/modules/storage/netapp/na_ontap_command.py

```diff
@@ -47,12 +54,14 @@
         password: "{{ admin password }}"
         command: ['network', 'interface', 'show']
         privilege: 'admin'
+        return_dict: true
 """
 
 RETURN = """
 """
 
 import traceback
+import xml.parsers.expat
```

Comment:
do we expect this module to be always available?

If not, could we do this under a try block, and only fails if return_dict is True ?

Reply:
I've put import into XML parse function. However, all netapp module users will have this module. It's used by NetApp SDK:

```
grep -R xml.parsers.expat /opt/netapp-sdk/lib/python/
/opt/netapp-sdk/lib/python/NetApp/NaServer.py:import xml.parsers.expat
/opt/netapp-sdk/lib/python/NetApp/NaServer.py:        p = xml.parsers.expat.ParserCreate()
/opt/netapp-sdk/lib/python/NetApp/NaServer.py:        p = xml.parsers.expat.ParserCreate()
``` 
Import error is handled (just in case)
```
ok: [lab.filer] => 
  cmd_out_parsed:
    changed: true
    failed: false
    msg:
      invoked_command: ldap client show -fields base-dn,ldap-servers
      result_value: 0
      status: ''
      stdout: XML parsing failed. Cannot import xml.parsers.expat!
      stdout_lines: []
      xml_dict: {}

```

Reply:
@lonico let me know if you like this approach more;.